### PR TITLE
Fix text excerpts sample for HTML / CSS example

### PR DESCRIPTION
**Pull request by 5im0n**
It's possible to create text excerpts with multiple line only in CSS. No JavaScript is needed.

## Presubmit checklist
- [X] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [X] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [X] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/get-started/flutter-for/web-devs.md [hunk 1]
```diff
@@ -1016,8 +1014,9 @@ and the `overflow` property for handling overflow text.
     padding: 16px;
     color: #ffffff;
     [[highlight]]overflow: hidden;
-    text-overflow: ellipsis;
-    white-space: nowrap;[[/highlight]]
+  display: -webkit-box;
```
Comment (domesticmouse) [inline src/get-started/flutter-for/web-devs.md:1017]: I'm curious why the change in indent

Comment (5im0n) [reply src/get-started/flutter-for/web-devs.md:1017]: Youps ! it's an IDE error !

**Comment by khanhnwin** [comment]
Once the indentation is fixed, LGTM!

**Review by khanhnwin — APPROVED** [review]
